tests/test_macos.py

```diff
@@ -45,3 +70,48 @@ def test_macos(params: dict[str, Any], func: str) -> None:
     expected = expected_map[func]
 
     assert result == expected
+
+
+@pytest.mark.parametrize(
+    "params",
+    [
+        pytest.param({}, id="no_args"),
+        pytest.param({"appname": "foo"}, id="app_name"),
+        pytest.param({"appname": "foo", "version": "v1.0"}, id="app_name_version"),
+    ],
+)
+@pytest.mark.parametrize("multipath", [pytest.param(True, id="multipath"), pytest.param(False, id="singlepath")])
+def test_macos_homebrew(mocker: MockerFixture, params: dict[str, Any], multipath: bool, func: str) -> None:
+    mocker.patch("sys.prefix", "/opt/homebrew/opt/python")
```

Comment:
Eh, rather than test all methods I'd prefer if you'd only test the ones that are actually impacted.

Reply:
@singingwolfboy the one resolving conversations is the reviewer, not the PR author, because only the reviewer can tell if you addressed the topic or not.

Reply:
Understood. I will not mark conversations as resolved anymore.